.pre-commit-config.yaml

```diff
@@ -18,4 +18,4 @@ repos:
         entry: ./scripts/format_yaml.sh
         language: python
         types: [ yaml ]
-        additional_dependencies: [ 'PyYAML<6' ]
+        additional_dependencies: [ 'PyYAML' ]
```

Comment:
I think `>6.0` prevented issues on the mac (very recently for me - within the last couple of weeks). fyi